app/assets/javascripts/users.js.erb

```diff
@@ -50,6 +39,37 @@ function initMap() {
   }
 }
 
+function displayTargets() {
+  $.get('/user_targets').done(function(data) {
+    $.each(data.user_targets, function(i, val) {
+      var targetPos = {
+        lat: val['lat'],
+        lng: val['lng']
+      }
+      showTarget(targetPos,val['radius'],val['topic'])
+    })
+  })
+}
+
+function showTarget(position,radius,topic){
```

Comment:
add spaces between the paramaters `function showTarget(position, radius, topic)`